Makefile

```diff
@@ -15,7 +15,7 @@ setup: pipenv venv
 
 .PHONY: pipenv
 pipenv:
-	pip install --user pipenv
+	pip3 install --user pipenv
```

Comment:
Maybe try pip3, with pip as a fallback on an exit error code?
```
pip3 install --user pipenv || pip install --user pipenv
```
As-is, it probably won't work on Windows. Python 3 installs the binaries as `python.exe` and `pip.exe`. Although the [`py` version selector](https://docs.python.org/3.6/using/windows.html#python-launcher-for-windows) may be helpful, I don't think it's available on Unix.

Reply:
As an alternative, installing python / pip and python3 / pip3 together will also work without modifying the make rule; pipenv will select Python3 for the venv regardless (as long as it's installed on the host machine).

Reply:
Ok, sounds good, I probably try the pip3/pip fallback, I think that seems the most flexible to me. I'll try setting up the Vagrant VM I have from scratch and see how that goes.